explainer.md

```diff
@@ -458,7 +458,8 @@ function onSessionStarted(session) {
 Some features recognized by the UA but not explicitly listed in these arrays will be enabled by default for a session. This is only done if the feature does not require a signal of [user intent](privacy-security-explainer.md#user-intent) nor impact performance or the behavior of other features when enabled. At this time, only the following features will be enabled by default:
 | Feature | Circumstances |
```

Comment:
@toji the preview doesn't render this table correctly for me, are you seeing that as well?

Reply:
Thanks. Just needed to add a line of whitespace above it.